README.md

```diff
@@ -1 +1,23 @@
-# portfolio
\ No newline at end of file
+# Project Name:portfolio
+This is my first project at Microverse.
+## Built With
+.HTML
+.CSS
+## Getting Started
+to implement and run this project you should have the following prerequisites  
+## Prerequisites
+Text editor
+browser like crome
+## Author
+ [Githab](https://github.com/settings/profile)
+[linkedin](https://www.linkedin.com/in/haftamu-desta-795791a1/)
+[tiwtter](https://twitter.com/DestaHftamu?t=NQ4ovkdWbsfsjh62NFEXFg&s=09)
```

Comment:
[OPTIONAL]
- [ ] Please try to follow the format of the (README template)[https://github.com/microverseinc/readme-template/blob/master/README.md] when writing your README